packages/google-tag-manager/plugin.js

```diff
@@ -6,6 +6,30 @@ class GTM {
     this.options = options
   }
   init() {
+
+    // insert scripts on client side if it not preseted on server side
```

Comment:
/cc @pimlie Isn't there any easier method for vue-meta checking scripts existence?